geopandas/array.py

```diff
@@ -518,6 +518,12 @@ def length(self):
         self.check_geographic_crs(stacklevel=5)
         return shapely.length(self._data)
 
+    @property
+    def count_coordinates(self):
+        out = np.empty(len(self._data), dtype="int64")
```

Comment:
```suggestion
        out = np.empty(len(self._data), dtype=np.int_)
```
will fix the windows CI failures (the default int width is platform dependent).